changelog/unreleased/bug-fixes/2734--relax-systemd-service-sandbox.md

```diff
@@ -0,0 +1,2 @@
+The systemd service for vast no longer fails if the home directory of the vast
```

Comment:
```suggestion
The systemd service no longer fails if the home directory of the VAST
```

Reply:
The user name is not capitalized, so I only took the other edit.